code/src/main/java/frc/robot/subsystems/SwerveWheel.java

```diff
@@ -124,11 +134,76 @@ public void setDesiredState(SwerveModuleState state) {
 
     SmartDashboard.putNumber(m_name + " Encoder Reading", m_turningEncoder.getRadians());
     SmartDashboard.putNumber(m_name + " Angle Target",
-        (state.angle.getRadians() % (2 * Math.PI) + (2 * Math.PI)) % (2 * Math.PI));
+        (rangeFix(state.angle.getRadians())));
     SmartDashboard.putNumber(m_name + " Turning PID output", m_turningPIDController.calculate(
-        m_turningEncoder.getRadians(), (state.angle.getRadians() % (2 * Math.PI) + (2 * Math.PI)) % (2 * Math.PI)));
+        m_turningEncoder.getRadians(), (rangeFix(state.angle.getRadians()))));
     // Calculate the turning motor output from the turning PID controller.
     m_driveMotor.set(driveOutput);
     m_turningMotor.set(turnOutput);
   }
+
+  /**
+   *
+   *
+   * @param state Desired state with speed and angle
+   * @return returns Velocity Output from pid
+   */
+ 
+
+
+     /**
+   *
+   *
+   * @param WheelMPs given speed of the wheel in MPS
+   * @return scaled voltage needed to attain specified speed
+   */
+  
+  public double mpsToVoltOutput (double WheelMps){
+
+    double secToMin = 60.0;
+    double WheelMpm = WheelMps*secToMin;
```

Comment:
local variables should use camel case, actually in Java everything except classes uses camel case.